css-positioning/relative.css

```diff
@@ -0,0 +1,41 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
```

Comment:
another thing you could do is set `position: relative` here instead of on all the ids. It does the same thing, since this selector also selects all the ids, just less to type out!

That way, all the id selectors do is set the height.